dashboard/config/packages/test/web_profiler.yaml

```diff
@@ -0,0 +1,6 @@
+web_profiler:
```

Comment:
I guess this is default generated configuration, but a profiler and logging into a proof of concept seem to go unused?

Reply:
(while the profiler can be useful in `journal` as performance-sensitive and using parallel requests to the API to build the page)